Mathlib/Algebra/Group/Commute/Defs.lean

```diff
@@ -155,19 +155,16 @@ theorem pow_pow (h : Commute a b) (m n : ℕ) : Commute (a ^ m) (b ^ n) :=
   (h.pow_left m).pow_right n
 -- `MulOneClass.toHasMul` vs. `MulOneClass.toMul`
 
--- Porting note: `simpNF` told me to remove the `simp` attribute
 @[to_additive]
```

Comment:
Can we make this one (and the below) `simp` now?

Reply:
Could I request that this is out of scope for this PR?

If someone wants this to be `@[simp]`, they can do that, but we can't get bogged down in investigating adding `@[simp]` to anything we touch the comments for.

Reply:
I'm not quite sure I follow the reasoning here. To my understanding, a comment like this is to track the tech debt that `simp` was removed. And so removing the comment and not restoring the attribute means we are keeping the tech debt, but removing the fact that it can be tracked. To me, losing the fact that these lemmas used to be `simp` (and thus that the `simp` set is now weaker) doesn't seem beneficial to mathlib. Could you clarify?